src/System.Text.Encoding/tests/Fallback/EncoderReplacementFallbackTests.cs

```diff
@@ -109,7 +109,8 @@ public void CreateFallbackBuffer_Fallback_InvalidSurrogateChars_ThrowsArgumentOu
             EncoderFallbackBuffer buffer = new EncoderReplacementFallback().CreateFallbackBuffer();
 
             Assert.Throws<ArgumentOutOfRangeException>("charUnknownHigh", () => buffer.Fallback('a', '\uDC00', 0));
-            Assert.Throws<ArgumentOutOfRangeException>("CharUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
+            Assert.Throws<ArgumentOutOfRangeException>(
+                RuntimeDetection.IsNetFramework ? "CharUnknownLow" : "charUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
```

Comment:
@tarekgh - the new/corrected names are the non-framework names, which have been verified to work with a local CLR build.  Linked companion issue is at the top.  The changes in this PR represent the only test failures (although at least one non-failure in a different library is probably due more to how the test was written/translated than avoiding a similar issue).